README.md

```diff
@@ -1,74 +1,101 @@
+
+  ```powershell
+  $Cert = New-SelfSignedCertificate -CertstoreLocation Cert:\LocalMachine\My -DnsName <<ip-address|fqdn-host-name>>
+
+  Export-Certificate -Cert $Cert -FilePath C:\temp\<<cert-name>>
+
+  Enable-PSRemoting -SkipNetworkProfileCheck -Force
+
+  # Check for HTTP listeners
+  dir wsman:\localhost\listener
+
+  # If HTTP Listeners exist, remove them
+  Get-ChildItem WSMan:\Localhost\listener | Where -Property Keys -eq "Transport=HTTP" | Remove-Item -Recurse
+
+  # If HTTPs Listeners don't exist, add one
+  New-Item -Path WSMan:\LocalHost\Listener -Transport HTTPS -Address * -CertificateThumbPrint $Cert.Thumbprint –Force
+
+  # This allows old WinRm hosts to use port 443
+  Set-Item WSMan:\localhost\Service\EnableCompatibilityHttpsListener -Value true
+
+  # Make sure an HTTPs inbound rule is allowed
+  New-NetFirewallRule -DisplayName "Windows Remote Management (HTTPS-In)" -Name "Windows Remote Management (HTTPS-In)" -Profile Any -LocalPort 5986 -Protocol TCP
+
+  # For security reasons, you might want to disable the firewall rule for HTTP that *Enable-PSRemoting* added:
+  Disable-NetFirewallRule -DisplayName "Windows Remote Management (HTTP-In)"
+  ```
+
+  - `ip-address` or `fqdn-host-name` can be used for the `DnsName` property in the certificate creation. It should be the name that the actions runner will use to call to the IIS server.
+  - `cert-name` can be any name.  This file will used to secure the traffic between the actions runner and the IIS server
 
 ## Example
 
 ```yml
-# TODO: Fill in the correct usage
+...
+
 jobs:
-  job1:
-    runs-on: [self-hosted]
-    steps:
-      - uses: actions/checkout@v2
+  stop-iis:
+   runs-on: [windows-2019]
+   env:
+      server: 'iis-server.domain.com'
+      pool-name: 'website-pool'
+      cert-path: './server-cert'
 
-      - name: Add the action here
-        uses: im-open/this-repo@v1.0.0
+   steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Create Web Site
+        uses: im-open/iis-site-create@v1.0.0
         with:
-          input-1: 'abc'
-          input-2: '123
-```
+          server: '${{ secrets.iis_server }}'
+          website-name: '${{env.WEBSITE_NAME}}'
+          app-pool-name: '${{ secrets.pool-name }}'
+          website-host-header: '${{env.WEBSITE_HOST_HEADER}}'
+          website-path: '${{env.DEPLOYMENT_FOLDER}}'
+          website-cert-path: '${{ env.WEBSITE_CERT_PATH}}'
+          website-cert-password: '${{ secrets.site_cert_password }}'
+          website-cert-friendly-name: '${{env.WEBSITE_CERT_FRIENDLY_NAME}}'
+          service-account-id: '${{secrets.iis_admin_user}}'
+          service-account-password: '${{secrets.iis_admin_password}}'
+          server-public-key: ${{ env.IIS_SERVER_CERT_PATH}}
```

Comment:
The ones that are using envs should be added to the section above.  And the envs above should be updated to reflect what's being used here like for the cert path and app pool name.